src/components/organisms/TimingSpace/TimingSpace.scss

```diff
@@ -0,0 +1,42 @@
+@import "scss/constants.scss";
+
+$create-button-width: 136px;
```

Comment:
This doesn't look correct, we don't have hardcoded button width anywhere. ButtonNG should have max-width imo.

Reply:
Well, the `<ButtonNG/>` component has `$max-allowed-button-width: 275px;` and it's needed to override.